source/more-info/unsupported/source_mods.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "Supervisor source modifications"
+description: "More information on why supervisor source modifications marks the installation as unsupported."
+---
+
+## The issue
+
+We can't support source modification which are untested and not approved by the core developers.
+There is no problem if you like to run your installation with own modification, but then you lost the
+official supported state of the installation.
```

Comment:
```suggestion
We can't support source modifications that are untested and not approved by the Home Assistant project.
Of course, it is no problem, you are allowed to run your installation with modification of your own; however, it will cause the loss of the official supported state of the installation.
```